### PR TITLE
Fix nested stanza order.

### PR DESCRIPTION
**Pull request by reitermarkus**
Needed for https://github.com/Homebrew/brew/pull/15223.

### CODE REVIEW

File: Casks/xit.rb [hunk 1]
```diff
@@ -22,14 +22,14 @@
...
-    depends_on macos: ">= :monterey"
-
-    app "Xit#{arch} #{version}/Xit.app"
-
     livecheck do
       url :url
       regex(/^v?(\d+(?:\.\d+)+(?:b\d+)?)$/i)
     end
+
+    depends_on macos: ">= :monterey"
```
Comment (reitermarkus) [inline Casks/xit.rb:30]: This seems useless.

File: Casks/tripmode.rb [hunk 2]
```diff
@@ -17,12 +17,12 @@
...
         verified: "tripmode-updates.ch/app/"
 
-    depends_on macos: ">= :big_sur"
-
     livecheck do
       url "https://tripmode-updates.ch/app/appcast-v#{version.major}.xml"
       strategy :sparkle
     end
+
+    depends_on macos: ">= :big_sur"
```
Comment (reitermarkus) [inline Casks/tripmode.rb:25]: This seems useless.

File: Casks/docker.rb [hunk 3]
```diff
@@ -6,23 +6,23 @@
...
            intel: "07b54cc6a2c61f7f37bfa8698449a9db857e4b94133e3614a42a9d8345bc561b"
 
-    depends_on macos: ">= :big_sur"
-
     livecheck do
       url "https://desktop.docker.com/mac/main/#{arch}/appcast.xml"
       strategy :sparkle
     end
+
+    depends_on macos: ">= :big_sur"
```
Comment (reitermarkus) [inline Casks/docker.rb:25]: This seems useless.

File: Casks/choosy.rb [hunk 4]
```diff
@@ -3,63 +3,63 @@
...
     sha256 "8d6a44b78ed256d6f502872fd1f62cf1f7fea877906bedddc5bbf26f93b6ea57"
 
-    depends_on macos: ">= :big_sur"
-
     livecheck do
       url "https://www.choosyosx.com/sparkle/feed"
       strategy :sparkle
     end
 
+    depends_on macos: ">= :big_sur"
```
Comment (reitermarkus) [inline Casks/choosy.rb:61]: This seems useless.

**Review by razvanazamfirei — APPROVED** [review]

**Comment by razvanazamfirei** [comment]
@reitermarkus, not sure what's going on with https://github.com/Homebrew/homebrew-cask/actions/runs/4906646219/jobs/8761662682?pr=146651#logs

**Comment by reitermarkus** [comment]
Livecheck seems to be timing out. Let's retry one more time.

**Pull request merged**